notification/alertstatus.go

```diff
@@ -5,6 +5,17 @@ type AlertStatus struct {
 	CallbackID string
 	AlertID    int
 	LogEntry   string
+
+	// Summary of the alert that this status is in regards to.
+	Summary string
+	// Details of the alert that this status is in regards to.
+	Details string
+
+	// OriginalStatus is the status of the first Alert notification to this Dest for this AlertID.
+	OriginalStatus SendResult
+
+	// Status is the most recent status of the Alert.
+	Status string
```

Comment:
can this be alert.Status instead of a string?

Reply:
That was my initial thought too, but that leads to an import cycle. So to keep things simple, I thought of having it as a string instead.